blog/12-25-22-best-practices-for-writing-smart-contracts-with-real-world-examples/best-practices-for-writing-smart-contracts-with-real-world-examples.md

```diff
@@ -0,0 +1,534 @@
+---
+title: Best practices for writing smart contracts with real world examples
+description: Showing some of the best practices for writing smart contracts.
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [solidity, celo, alfajores, smart-contract]
+hide_table_of_contents: true
+slug: /tutorials/best-practices-for-writing-smart-contracts-with-real-world-examples
+---
+
+![header](images/3.png)
+
+# Introduction
```

Comment:
Only use `##` in your articles. `#` is only used once for the title, which comes from docusaurus.
Keeping only one `#` helps with better SEO.